packages/core/src/cart/cart-source.ts

```diff
@@ -0,0 +1,3 @@
+export enum CartSource {
```

Comment:
Do we need to duplicate `CartSource` in core package?
Why we can't use it from payment-integration-api package?

Reply:
+